files/default/psql-env-ckan.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+if [ "$CKAN_PROPERTY" = "" ]; then
+  CKAN_PROPERTY="sqlalchemy[.]url"
+fi
+CONNECTION_STRING=$(grep "$CKAN_PROPERTY" /etc/ckan/default/production.ini |awk -F '//' '{print $2}')
+export PGUSER=$(echo "$CONNECTION_STRING" | awk -F ':' '{print $1}')
```

Comment:
and what about dba root user but this still fills in the pghost

Reply:
Batch servers, by design, do not have access to the DBA credentials. If you want to perform DBA operations, you need to go to SSM Parameter Store and look up the name and password yourself. At that point you're probably implementing a change request that should have details like the host prefilled anyway.